pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
And I think the first one is plain wrong (in the general case) :-) We don't know what the next version will be. Maybe they skip a version, add a new sub-minor version, or jump to the next major version?

+1 for a PR that updates the manual with this corner case: version number scheme for unreleased versions of a projects that _do_ have releases.